napari/util/misc.py

```diff
@@ -58,6 +59,135 @@ def is_rgb(shape):
         return False
 
 
+def is_pyramid(data):
+    """If data is a list of arrays of decreaing size.
+    """
+    if isinstance(data, list):
+        size = [np.prod(d.shape) for d in data]
+        return np.all(size[:-1] > size[1:])
+    else:
+        return False
+
+
+def should_be_pyramid(shape):
+    """Check if any data axes needs to be pyramided
```

Comment:
I think the correct verb is pyramidified. =P

Reply:
love it - changed